scripts/deploy_gke.sh

```diff
@@ -87,6 +91,18 @@ if [[ $PROJECT_ID != "" ]]; then
   cd $ROOT/deploy/overlays
   cp custom_kustomization.yaml.tpl kustomization.yaml
   sed -i '' "s/<PROJECT_ID>/$PROJECT_ID/g" kustomization.yaml
+  export PROJECT_ID=$PROJECT_ID
+  yq eval -i '.project = env(PROJECT_ID)' ../base/custom_bigtable_info.yaml
+  yq eval -i '.instance = "dc-graph"' ../base/custom_bigtable_info.yaml
```

Comment:
no need to set instance?

Reply:
It's an empty file right now. Not sure if the emptiness was intended so it was set dynamically.

Reply:
Ok, then should set the instance.

Reply:
done.